packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -1,132 +1,90 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import { LitElement, html, nothing } from 'lit';
+import {LitElement, html, nothing} from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import '@conversionxl/cxl-lumo-styles';
-import '@vaadin/progress-bar';
 import './cxl-dashboard-notification';
 import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
-import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
+
 import cxlDashboardHeaderStyles from '../styles/cxl-dashboard-header-css.js';
+import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
 
 @customElement('cxl-dashboard-header')
 export class CXLDashboardHeaderElement extends LitElement {
   static get styles() {
     return [cxlDashboardHeaderStyles];
   }
 
-  @property({ type: String }) title = 'Hello';
-
-  @property({ type: String }) subtitle = 'My dashboard';
-
-  @property({ type: String }) name = '';
-
   @property({ type: Number, attribute: 'notification-count' }) notificationCount = 0;
 
   @property({ type: String }) notificationTitle = "What's new in CXL";
 
   @property({ type: Object }) notificationData = null;
 
-  @property({ type: String, attribute: 'last-course-title' }) lastCourseTitle = '';
-
-  @property({ type: String, attribute: 'last-course-link' }) lastCourseLink = '';
-
-  @property({ type: String }) lastCourseSubtitle = 'Continue where you left off';
-
-  @property({ type: Number, attribute: 'progress' }) progress = 0;
-
-  @property({ type: Number, attribute: 'lessons-completed' }) lessonsCompleted = 0;
-
-  @property({ type: Number, attribute: 'lessons-total' }) lessonsTotal = 0;
-
-  @property({ type: String }) cta1 = '';
+  @property({ type: String }) subtitle = 'My dashboard';
 
-  @property({ type: String }) cta1Link = '';
+  @property({ type: String }) title = 'Hello';
 
-  @property({ type: String }) cta2 = '';
+  @property({ type: String }) name = '';
 
-  @property({ type: String }) cta2Link = '';
+  @property({ type: Boolean }) hasCompletedStats = false;
```

Comment:
I think `show` is better in the context of (not)showing a piece of component.

```suggestion
  @property({ type: Boolean }) showCompletedStats = false;
```